static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -44,14 +45,22 @@ export const Showcase = ({cards, setSelectedFilterOptions}: ShowcaseProps) => {
 
   return (
     <div>
-      <Byline>
-        Showcase resources: click to filter the resources to a pathogen
-      </Byline>
       <ShowcaseContainer className={!isExpandable ? "" : isExpanded ? "expanded" : "collapsed"} $expandedHeight={cardsContainerHeight}>
         <CardsContainer ref={cardsContainerRef}>
-          {cards.map((el) => (
-            <ShowcaseTile card={el} key={el.name} setSelectedFilterOptions={setSelectedFilterOptions}/>
-            ))}
+          {cards.map((el) => {
+            if (el.filters && el.url) {
+              throw new Error("Cards used in this component cannot have both `filters` and `url`.");
+            }
+            else if (el.filters) {
+              return <FilterShowcaseTile card={el} key={el.name} setSelectedFilterOptions={setSelectedFilterOptions}/>
+            }
+            else if (el.url) {
+              return <UrlShowcaseTile card={el} key={el.name} />
+            }
+            else {
+              throw new Error("Cards used in this component must have `filters` or `url`.");
```

Comment:
\<nit level=extreme>
```suggestion
              throw new Error("Cards used in this component must have one of `filters` or `url`.");
```

Reply:
Sorry I forgot to update the PR – this change is from #901 and has since been removed (see https://github.com/nextstrain/nextstrain.org/pull/901#discussion_r1633981200)